docs/Text.md

```diff
@@ -2,6 +2,12 @@
 
 Use the `<Text />` component to control font size, weight, alignment, and color.
 
+###### Note: To use the `Montserrat` font-family in you project, don't forget to include the following code inside your html `<head>`
```

Comment:
👍

Reply:
For performance reasons, we'll probably want to investigate different font loading strategies and have that built into the boilerplate at some point

Reply:
cc @BeniCheni

Reply:
webfontloader is good. there's probably some opportunities for use to do things like cache the font in localStorage/etc – not sure what current best practices are around this sort of thing

Reply:
I have never tried localStorage caching but I have heard many times that it's a great hack for perf optimization. Also, a great article about loading and rendering fonts here: https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/webfont-optimization#optimizing_loading_and_rendering